deployment.go

```diff
@@ -44,6 +46,14 @@ type DeploymentStep struct {
 	App    string `json:"app"`
 }
 
+// DeploymentStepV1 is a Marathon v1.X-compatible DeploymentStep
+type DeploymentStepV1 struct {
+	Actions []struct {
+		Type string `json:"type"`
```

Comment:
I had seen that however that change does not exist in the current release (v1.1.1) which I am attempting to fix and instead be included with the next [release](https://github.com/mesosphere/marathon/milestones/0.16.0). I feel it is best to wait until that version is released, and then update since this PR aims to explicitly support **v1.1.1**.